src/routes/RemediationDetailsComponents/ActionsContent.js

```diff
@@ -0,0 +1,209 @@
+    }
+    return allIssues.filter((issue) => issue.description.includes(nameFilter));
+  }, [allIssues, nameFilter]);
+  const start = params?.offset || 0;
+  const end = (params?.limit || 10) + start;
+  const pageOfIssues = filteredIssues.slice(start, end);
+
+  const columnsWithSystemsButton = useMemo(() => {
+    return columns.map((col) => {
+      if (col.exportKey === 'system_count') {
+        return {
+          ...col,
+          renderFunc: (_data, _id, { systems, description }) => (
+            <Button
+              variant="link"
+              onClick={() => {
+                setActionToShow(description);
+                setSystemsToShow(systems);
+                setIsSystemsModalOpen(true);
+              }}
+            >
+              {`${systems.length} system${systems.length !== 1 ? 's' : ''}`}
+            </Button>
+          ),
+        };
+      }
+      return col;
+    });
+  }, []);
+  return (
+    <section
+      className={'pf-v5-l-page__main-section pf-v5-c-page__main-section'}
+    >
+      {isSystemsModalOpen && (
+        <SystemsModal
+          isOpen={isSystemsModalOpen}
+          onClose={() => setIsSystemsModalOpen(false)}
+          systems={systemsToShow}
+          actionName={actionToShow}
+        />
+      )}
+      {isDeleteModalOpen && (
+        <ConfirmationDialog
+          isOpen={isDeleteModalOpen}
+          title={`Remove action(s)`}
+          text="You will not be able to recover this/these action(s)"
+          confirmText="Remove action(s)"
+          onClose={(confirm) => {
+            setIsDeleteModalOpen(false);
+            if (confirm) {
+              const chopped = isBulkDelete ? currentlySelected : action;
+              handleDelete(chopped)
+                .then(() => {
+                  dispatchNotification({
+                    title: `Succesfully deleted action(s)`,
+                    description: '',
+                    variant: 'success',
+                    dismissable: true,
+                    autoDismiss: true,
+                  });
+                  callbacks?.current?.resetSelection();
+                  refetch();
+                  setIsDeleteModalOpen(false);
+                  setIsBulkDelete(false);
+                })
+                .catch(() => {
+                  dispatchNotification({
+                    title: `Failed to delete action(s)`,
+                    description: '',
+                    variant: 'danger',
+                    dismissable: true,
+                    autoDismiss: true,
+                  });
+                });
+            }
+          }}
+        />
+      )}
+      <RemediationsTable
+        aria-label="ActionsTable"
+        ouiaId="ActionsTable"
+        items={pageOfIssues}
+        total={filteredIssues.length}
+        columns={[...columnsWithSystemsButton]}
+        filters={{
+          filterConfig: [...actionNameFilter],
+        }}
+        options={{
+          onSelect: () => '',
```

Comment:
Why would `onSelect` return a string here?

Reply:
Ill add a note because without context this is very odd, this is a known issue in AsyncTableTools, and will be fixed once the PR for it is merged and we convert a lot of the code here to be used from the new AsyncTableTools package